core/EE_System.core.php

```diff
@@ -1105,17 +1112,24 @@ public function perform_activations_upgrades_and_migrations()
     }
 
 
-
     /**
-     *    load_CPTs_and_session
-     *
-     * @access public
-     * @return    void
+     * @return void
+     * @throws DomainException
      */
     public function load_CPTs_and_session()
     {
         do_action('AHEE__EE_System__load_CPTs_and_session__start');
-        // register Custom Post Types
+        /** @var EventEspresso\core\domain\services\custom_post_types\RegisterCustomTaxonomies $register_custom_taxonomies */
+        $register_custom_taxonomies = $this->loader->getShared(
+            'EventEspresso\core\domain\services\custom_post_types\RegisterCustomTaxonomies'
+        );
+        $register_custom_taxonomies->registerCustomTaxonomies();
+        /** @var EventEspresso\core\domain\services\custom_post_types\RegisterCustomPostTypes $register_custom_post_types */
+        $register_custom_post_types = $this->loader->getShared(
+            'EventEspresso\core\domain\services\custom_post_types\RegisterCustomPostTypes'
+        );
+        $register_custom_post_types->registerCustomPostTypes();
+        // register Custom Post Types and Taxonomies
         $this->loader->getShared('EE_Register_CPTs');
```

Comment:
why are you still calling the deprecated class here?

Reply:
we still have lots of code referencing that class in both core and add-ons and there may also be third party code referencing it, so it needs to be loaded. It no longer performs any CPT registration though.